src/pattern-library/components/patterns/LayoutFoundations.js

```diff
@@ -1,159 +1,15 @@
-import classnames from 'classnames';
 import { useState } from 'preact/hooks';
 
-import { LabeledButton } from '../../../';
+import { Actions, Card, LabeledButton } from '../../../';
 import Library from '../Library';
 
-function SquareBlock() {
-  return (
-    <div
-      className="hyp-u-bg-color--grey-4"
-      style={{ width: '2rem', height: '2rem' }}
-    />
-  );
-}
-
-/**
- * Render an example of vertical or horizontal spacing between elements.
- *
- * @param {object} options
- *   @param {'horizontal'|'vertical'} options.direction
- *   @param {number} options.size - Spacing unit size, 0 - 9
- *   @param {boolean} [options.defaultSize] - Is this the "default" spacing for
- *     this orientation?
- */
-function SpacingDemo({ direction, size, defaultSize = false }) {
-  const layoutClass =
-    direction === 'vertical' ? 'hyp-u-layout-column' : 'hyp-u-layout-row';
-  const baseClass =
-    direction === 'vertical'
-      ? 'hyp-u-vertical-spacing'
-      : 'hyp-u-horizontal-spacing';
-  const sizeClass = `${baseClass}--${size}`;
-  return (
-    <div className={classnames(layoutClass, baseClass)}>
-      {defaultSize ? (
-        <div>
-          <strong>{size}*</strong>
-        </div>
-      ) : (
-        <div>{size}</div>
-      )}
-      <div className={classnames(layoutClass, sizeClass, 'hyp-u-border')}>
-        <SquareBlock />
-        <SquareBlock />
-        <SquareBlock />
-        <SquareBlock />
-        <SquareBlock />
-      </div>
-      {direction === 'horizontal' && (
-        <div>
-          <code>.{sizeClass}</code>
-        </div>
-      )}
-    </div>
-  );
-}
-
 export default function LayoutFoundations() {
   const [showExample1, setShowExample1] = useState(false);
   const [showExample2, setShowExample2] = useState(false);
   const [showExample3, setShowExample3] = useState(false);
-  const scaleUnits = [
```

Comment:
We no longer need to build and maintain our own scaled spacing system, thank you tailwind. See Tailwind docs instead :).